scylla-gdb.py

```diff
@@ -62,6 +62,13 @@ def get_field_offset(gdb_type, name):
         if field.name == name:
             return int(field.bitpos / 8)
 
+@functools.cache
```

Comment:
nit, would be better to have two empty lines between top-level functions instead of one empty line.